qulice-maven-plugin/src/it/dependency-violations/LICENSE.txt

```diff
@@ -1,2 +1,27 @@
-Some text.
```

Comment:
@simonjenga This file is deliberately designed to test whether the violation check is raised, don't change it!!!